builders/manage/developer-portal/smart-contracts-creation-filter.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Smart EVM - Whitelist Smart Contracts Deployment
+description: Learn how to use Sudo to whitelist smart contracts deployment for your Smart EVM Tanssi appchain, increasing overall security.
+---
+
+# Smart EVM - Whitelist Smart Contracts Deployment
+
+## Introduction {: #introduction }
+
+EVM-compatible Tanssi appchains benefit from a unique feature: the appchain governor can define which accounts are authorized to deploy smart contracts, forbidding the action for any other non-whitelisted account.
+
+This feature brings several key benefits that might be a great fit for different use cases or contexts. Some of those benefits are:
+
+- **Enhanced Security** - by restricting deployment to trusted accounts, the risk of deploying malicious or vulnerable smart contracts is reduced
+- **Quality Assurance** - known and vetted accounts can be required to follow specific coding standards and undergo thorough testing before deployment
+- **Regulatory Compliance** - uses cases that are highly regulated can limit deployment to ensure that smart contracts meet legal and compliance requirements
+- **Spam and Abuse Prevention** - prevent bad actors from deploying large numbers of unnecessary or harmful contracts
+
+In this guide, you'll learn how to use the Sudo account to manage the whitelisted accounts that will be able to deploy smart contracts on your appchain.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
```

Comment:
These prerequisites are not in a snippet right?

Reply:
Not yet, but definitely should.
I'll create a ticket to revisit these guides, create the snippets and update the screenshots